src/js/components/__tests__/__snapshots__/README-test.js.snap

```diff
@@ -4740,7 +4735,7 @@ string
 Required. Array of objects containing a value. The array should already be
       sorted from largest to smallest value. The caller can put other
       properties in the object. The children function will be called to
-      render the contents of each value. Defaults to \`[]\`.
+      render the contents of each value.
```

Comment:
where did this change come from? can you clean up your branch and remove this snapshot, unless you think it should be here.

Reply:
@ShimiSun if possible can you please review this ?